smooth-scroll/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for smooth-scroll 10.1.0
```

Comment:
`npm info smooth-scroll` gives me [enkidevs/smooth-scroll](https://github.com/enkidevs/smooth-scroll), but this is for [cferdinandi/smooth-scroll](https://github.com/cferdinandi/smooth-scroll). NPM packages win, so this one should be renamed. I think `cferdinandi-smooth-scroll` would work.